t/server-iterate-singlestep.t

```diff
@@ -24,15 +25,30 @@ is($client->{client}->connect_async(
     $data
 ), STATUSCODE_GOOD, "connect async");
 
-while ($client->{client}->getState() != CLIENTSTATE_SESSION) {
-    $client->{client}->run_iterate(0);
+my $i;
+for ($i = 50; $i > 0; $i--) {
+    my $sc = $client->{client}->run_iterate(0);
     $server->step();
+
+    if ($client->{client}->getState() == CLIENTSTATE_SESSION) {
+    	pass "server: client session established";
```

Comment:
The "server: " is used to indicate a log from Test::Server.  This is test code, no prefix necessary.

Reply:
Fixed.